### PR TITLE
Downgrade Rust Container

### PR DESCRIPTION
**Pull request by TruongQuangSB**
-Downgrade Rust Container to 1.65.

### CODE REVIEW

File: docker/rust/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,4 @@
-FROM rust:1.71
+FROM rust:1.65
```
Comment (TruongQuangSB) [inline docker/rust/Dockerfile:1]: Somehow cef-browser doesn't work with rust container version after 1.65

File: docker/rust/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,4 @@
...
 
 RUN apt-get update && \
-    apt-get install --assume-yes --no-install-recommends gcc-mingw-w64 libz-mingw-w64-dev
-
-RUN rustup target add x86_64-pc-windows-gnu && \
```
Comment (TruongQuangSB) [inline docker/rust/Dockerfile:6]: With this version must rust setup move to browser workflows

**Comment by TruongQuangSB** [comment]
After commit https://github.com/eclipse-set/browser/commit/aad4a68f4956fe521ee810a7e6e02c2d8e2f96f4, we don't need rust container mode

File: .github/workflows/build-docker-images.yml [hunk 3]
```diff
@@ -23,11 +23,3 @@ jobs:
           registry: ${{ env.REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
```
Comment (pstuecker) [inline .github/workflows/build-docker-images.yml:25]: We can remove the entire workflow here

**Review by pstuecker — CHANGES_REQUESTED** [review]

**Review by pstuecker — APPROVED** [review]

**Pull request merged**